jerry-core/ecma/builtin-objects/ecma-builtin-arraybuffer-prototype.c

```diff
@@ -0,0 +1,141 @@
+/* Copyright 2016 Intel Corporation
```

Comment:
Do you mean it should be `Copyright JS Foundation and other contributors, http://js.foundation` as the **only copyright notice for all files in jerryscript**?